templates/registration/index.html.twig

```diff
@@ -0,0 +1,128 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Register{% endblock %}
+
+{% block body %}
+<style>
```

Comment:
don't use css directly in template files

Reply:
noted